disk/src/com/treode/disk/messages/package.scala

```diff
@@ -38,3 +39,9 @@ package messages {
   case class AlreadyDraining (drive: Path) extends Message {
     def en = s"Already draining: ${quote (drive)}"
   }}
+
+package object messages {
+
+  implicit def stringToPath (path: String): Path =
```

Comment:
Where can I define it in SpecTools in order to be able to access it in DrivesAttachHandlerSpec? I tried putting an `implicit def` inside `trait SpecTools` but it doesn't seem to be in scope.